backend/typescript/services/interfaces/campService.ts

```diff
@@ -18,6 +18,14 @@ interface ICampService {
    */
   getCamps(): Promise<GetCampDTO[]>;
 
+  /**
+   * Get camp with the specified campId
+   * @param campId camp's id
+   * @returns a CampDTO
+   * @throws Error if camper retrieval fails
+   */
+  getCampById(campId: string): Promise<CampDTO>;
```

Comment:
update return type here since we're populating the questions and sessions fields!
```suggestion
  getCampById(campId: string): Promise<GetCampDTO>;
```